### PR TITLE
feat: Find from cadastral registry function implemented

### PR DESCRIPTION
**Pull request by JCruiz15**
- [X] Create a function that directly reads a cadastral registry and returns geometry and metadata.
- [X] Ensure that the control characters match the expected ones.
- [x] Create a suite of integration tests with different cadastral records.

### CODE REVIEW

File: sigpac_tools/utils.py [hunk 1]
```diff
@@ -92,3 +95,131 @@ def findCommunity(province_id: int) -> int:
...
+        "parcel": int(reg_par),
+        "id_inm": int(reg_id),
+        "control": reg_control,
+    }
+
+
+def validate_cadastral_registry(reference: str) -> None:
+    """Validate the cadastral reference
+
+    Given a cadastral reference, it validates if the reference is correct or not by comparing the present control characters with the calculated expected ones.
```
Comment (jfaldanam) [inline sigpac_tools/utils.py:159]: Link to the documentation we found of this, even if it is not official.

**Review by jfaldanam — APPROVED** [review]
LGTM, minor changes

File: sigpac_tools/__main__.py [hunk 2]
```diff
@@ -199,6 +211,19 @@ def main():
...
             logger.info(f"Metadata:\n{json.dumps(metadata, indent=2)}")
             return metadata
+        case "find":
+            from sigpac_tools.find import find_from_cadastral_registry
+            import json
+
+            registry = args.registry
+            geom, metadata = find_from_cadastral_registry(registry)
+            logger.info(
+                f"Geometry for cadastral registry {registry}:\n{json.dumps(geom, indent=2)}"
```
[the remark this answers is not in the document]
Comment (JCruiz15) [reply sigpac_tools/__main__.py:221]: Geom and metadata are different things

[the remark this answers is not in the document]
Comment (jfaldanam) [reply sigpac_tools/__main__.py:221]: :zany_face:

**Pull request merged**